### PR TITLE
Add constants for the property keys

### PR DESCRIPTION
**Pull request by DilshanSenarath**
### Related Issues
- https://github.com/wso2/product-is/issues/18028

### Related PRs
- https://github.com/wso2/identity-apps/pull/4802

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: features/admin.extensions.v1/components/identity-providers/constants/swe-constants.ts [hunk 1]
```diff
@@ -44,4 +44,13 @@ export class SIWEConstants {
...
         PROFILE: "profile"
     };
+
+    /**
+     * Properties keys for the SWE authenticator.
+     */
+    public static readonly SWE_AUTHENTICATOR_PROPERTY_KEYS: Record<string, string> = {
+        CALLBACK_URL: "callbackUrl",
+        CLIENT_ID: "ClientId",
+        CLIENT_SECRETE: "ClientSecret"
```
Comment (pavinduLakshan) [inline features/admin.extensions.v1/components/identity-providers/constants/swe-constants.ts:54]:
```suggestion
        CLIENT_SECRET: "ClientSecret"
```

File: features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx [hunk 2]
```diff
@@ -248,21 +249,23 @@ export const SIWEAuthenticationProviderCreateWizard: FunctionComponent<
             identityProvider.federatedAuthenticators.authenticators[ 0 ].properties =
                 identityProvider?.federatedAuthenticators?.authenticators[ 0 ]?.properties?.filter(
                     (item: CommonPluggableComponentPropertyInterface) =>
-                        item.key !== "ClientId" && item.key !== "ClientSecret" && item.key !== "callbackUrl"
+                        item.key !== SIWEConstants.SWE_AUTHENTICATOR_PROPERTY_KEYS.CLIENT_ID
+                            && item.key !== SIWEConstants.SWE_AUTHENTICATOR_PROPERTY_KEYS.CLIENT_SECRETE
```
Comment (pavinduLakshan) [inline features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx:253]:
```suggestion
                            && item.key !== SIWEConstants.SWE_AUTHENTICATOR_PROPERTY_KEYS.CLIENT_SECRET
```

File: features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx [hunk 3]
```diff
@@ -248,21 +249,23 @@ export const SIWEAuthenticationProviderCreateWizard: FunctionComponent<
...
             identityProvider.federatedAuthenticators.authenticators[ 0 ].properties = [
                 ...identityProvider.federatedAuthenticators.authenticators[ 0 ].properties,
                 {
-                    "key": "ClientId",
+                    "key": SIWEConstants.SWE_AUTHENTICATOR_PROPERTY_KEYS.CLIENT_ID,
                     "value": values.clientId.toString()
                 },
                 {
-                    "key": "ClientSecret",
+                    "key": SIWEConstants.SWE_AUTHENTICATOR_PROPERTY_KEYS.CLIENT_SECRETE,
```
Comment (pavinduLakshan) [inline features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx:264]:
```suggestion
                    "key": SIWEConstants.SWE_AUTHENTICATOR_PROPERTY_KEYS.CLIENT_SECRET,
```

File: features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx (hunk 2 above)
Comment (DilshanSenarath) [reply features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx]: https://github.com/wso2/identity-apps/pull/6047/commits/86f8fcbf4c460580c90cd25858d4d181527a53b1

File: features/admin.extensions.v1/components/identity-providers/constants/swe-constants.ts (hunk 1 above)
Comment (DilshanSenarath) [reply features/admin.extensions.v1/components/identity-providers/constants/swe-constants.ts]: https://github.com/wso2/identity-apps/pull/6047/commits/86f8fcbf4c460580c90cd25858d4d181527a53b1

File: features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx (hunk 3 above)
Comment (DilshanSenarath) [reply features/admin.extensions.v1/identity-provider-templates/templates/swe/wizards/swe-authentication-provider-create-wizard.tsx]: https://github.com/wso2/identity-apps/pull/6047/commits/86f8fcbf4c460580c90cd25858d4d181527a53b1

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**